common.mk

```diff
@@ -268,11 +270,12 @@ endif
 %.bin: %.elf
 	$(call cp_file,$<,$@)
 
-%.elf: $(CRTOBJFILES) $(FINOBJFILES)
```

Comment:
Is there a reason not to add `$(ARCHIVES)` here, allowing the `CC` line below to keep using `$^` ?